cluster/dragon/dragon.go

```diff
@@ -904,39 +909,61 @@ func (d *Dragon) tableExists(queryExec common.SimpleQueryExec, id uint64) (bool,
 	return rows.RowCount() != 0, nil
 }
 
-// Currently the number of shards in the cluster is fixed. We must make sure the user doesn't change the number of shards
-// in the config after the cluster has been created. So we store the number in the database and check
-func (d *Dragon) checkConstantShards(expectedShards int) error {
-	log.Debugf("Checking constant shards: %d", expectedShards)
+func (d *Dragon) getConfigProperty(property string) ([]byte, error) {
 	key := table.EncodeTableKeyPrefix(common.LocalConfigTableID, 0, 16)
-	propKey := []byte("num-shards")
+	propKey := []byte(property)
 	key = common.AppendUint32ToBufferBE(key, uint32(len(propKey)))
 	key = append(key, propKey...)
-	v, err := d.LocalGet(key)
+	return d.LocalGet(key)
+}
+
+func (d *Dragon) setConfigProperty(property string, v []byte) error {
```

Comment:
Property is not used here, so the key will be generated incorrectly.
Can you add a test in dragon_integration_test.go for this?

Reply:
Hi @mohsalsaleem I think we're almost there. If you can add a test, them we can get this merged!